dandiapi/api/management/commands/import_dandisets.py

```diff
@@ -1,14 +1,65 @@
 from urllib.parse import urljoin
+from uuid import uuid4
 
+from django.core.files.uploadedfile import SimpleUploadedFile
 from django.db import transaction
 import djclick as click
 import requests
 
-from dandiapi.api.models import Dandiset, Version
+from dandiapi.api.models import Asset, AssetBlob, Dandiset, Version, ZarrArchive
 
 
 @transaction.atomic
-def import_versions_from_response(api_url: str, version_api_response: dict, dandiset: Dandiset):
+def import_assets_from_response(api_url: str, asset_api_response: dict, version: Version):
+    uploaded_file = SimpleUploadedFile(name='foo/bar.txt', content=b'A' * 20)
+    etag = '76d36e98f312e98ff908c8c82c8dd623-0'
+    asset_blob = AssetBlob.objects.get_or_create(
+        etag=etag, defaults=dict(blob_id=uuid4(), blob=uploaded_file, etag=etag, size=20)
+    )[0]
+
+    for result in asset_api_response['results']:
+        blob = result.get('blob')
+        zarr = result.get('zarr')
+
+        if blob:
+            asset_metadata = requests.get(
+                urljoin(
+                    api_url,
+                    f'/api/dandisets/{version.dandiset.identifier}/versions/'
+                    f'{version.version}/assets/{result["asset_id"]}/',
+                ),
+                {'page_size': 1000},
+            ).json()
```

Comment:
Is this level of fidelity required for these closed assets? I'd think the default populated metadata would be sufficient, since the underlying data doesn't match anyway. This would half the amount of requests made for non-zarr assets.

Reply:
Good point, none of the metadata would be valid anyway :stuck_out_tongue:, I updated it to use the same minimal asset metadata blob for every asset.